packages/trader/src/Stores/Modules/Cashier/cashier-store.js

```diff
@@ -571,7 +556,17 @@ export default class CashierStore extends BaseStore {
         await this.onMountCommon();
         await BinarySocket.wait('website_status');
 
-        if (!this.config.account_transfer.accounts_list.length) {
+        // check if some balance update has come in since the last mount
+        const has_updated_account_balance = this.config.account_transfer.has_no_accounts_balance &&
+            Object.keys(this.root_store.client.active_accounts).find(account =>
+                !this.root_store.client.active_accounts[account].is_virtual &&
+                this.root_store.client.active_accounts[account].balance
+            );
+        if (has_updated_account_balance) {
+            this.setHasNoAccountsBalance(false);
```

Comment:
** can't has_no_accounts_balance be a computed property? since it's derived from the account balance

Reply:
@oskar-binary right now it's based on the transfer_between_accounts response, so we set it to false here but further down it will check if client canDoAccountTransfer and based on the transfer_between_accounts API call sets it to true if none of the accounts returned there have any balance. here I'm just setting it to false to get the code in the jsx file to show the loader instead of showing the no balance screen.